docs/testkube-pro/articles/environment-management.md

```diff
@@ -1,19 +1,19 @@
 # Environment Management
 
-Environment in Testkube is related to the Testkube agent, which is responsible for sending
-test insights to Testkube Pro and for managing your Kubernetes related cluster resources.
+A Testkube environment is managed by a Testkube Agent running in your Kubernetes Cluster - read more [here](../articles/architecture.md).
```

Comment:
please put this line in a note and add the line "An environment is a grouping of users, workflows, results, triggers, and webhooks." before it

Reply:
Updated.